### PR TITLE
Make last train -E optional

### PR DESCRIPTION
**Pull request by RomainDallet**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by lecorguille** [comment]
@RomainDallet Can you bump the wrapper version?

File: tools/last/lasttrain.xml [hunk 1]
```diff
@@ -74,7 +73,7 @@
             <!-- Training options missing  -->
             <section name="align_opt" title="Alignment options" expanded="true">
                 <param argument="-D" type="integer" value="1000000" label="Query letters per random alignment"/>
-                <param argument="-E" type="integer" value="10" label="Maximum expected alignments per square giga"/>
+                <param argument="-E" type="integer" value="" optional="true" label="Maximum expected alignments per square giga"/>
```
Comment (bernt-matthias) [inline tools/last/lasttrain.xml:76]: A test case that asserts that `-E ...` is not set on the command line would be nice.

Comment (bernt-matthias) [reply tools/last/lasttrain.xml:76]: The error caused by the diff to the test data should disappear if you set `-E` to the old default of 10 ...

Comment (bernt-matthias) [reply tools/last/lasttrain.xml:76]: Can you add some statement to the help on what happens if this parameter is left empty?

**Comment by RomainDallet** [comment]
@bernt-matthias Do you have any idea why the error persists? The tests have passed on my computer.

**Comment by bernt-matthias** [comment]
The first test fails because the test data `last_train_E.txt` is not yet commited.

**Comment by bernt-matthias** [comment]
I also guess that you mixed up the two test output files, or?

**Comment by RomainDallet** [comment]
> The first test fails because the test data `last_train_E.txt` is not yet commited.

Oops, yes I had not seen this oversight.

**Review by bernt-matthias — APPROVED** [review]

**Comment by bernt-matthias** [comment]
Fine from my side. Any objections against merging @lecorguille ?

**Comment by lecorguille** [comment]
@bernt-matthias It's all good for me!

**Comment by bernt-matthias** [comment]
Thanks @RomainDallet

**Pull request merged**